spec/spec_helper.rb

```diff
@@ -23,6 +23,7 @@
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 # include the manageiq-gems-pending matchers
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
```

Comment:
Is this really necessary? There's already a `/shared` with `models/floating_ip.rb`. How is that currently required now? (or is it unused?)